### ISSUE TITLE
An 3x3 overloaded ame can produce more power than a stable than 3x6 AME

### ISSUE THREAD

**Issue by nikthechampiongr** [space-wizards/space-station-14#23064]
## Description

I am not sure if this is intended or not so I'll just make the issue. A slightly overloaded AME with one core can produce the same amount of power as an AME with 4. The AME also takes a very long time to actually become dangerous now, and it can't even blow up in the first 30 minutes mechanically.
**Reproduction**

Overload a 3x3 ame and laugh as it powers a station too big for it to ever be able to power.

**Comment by IProduceWidgets** [comment]
plant power sink > overload AME to 1 bajillion> profit?

**Comment by deltanedas** [comment]
only has 1mw load max

**Comment by daerSeebaer** [comment]
To better understand the question, because I think some things behave differently:
1. Where is it specified that an AME can't blow up in the first 30 minutes mechanically? There is an open issue requesting this, but I don't think that has yet been implemented / is currently being worked on?
2. If I mathed correctly, a 1-core AME overloaded to 320kW (values pre bandaid-merge) will likely explode after somewhere between 20 and 35 minutes. Do you think that is too long? Also this time has not been changed by the last lower-damage PR.
3. Where does it say that an AME has a 1MW load maximum?

All of this assuming the current bandaid with absurd high AME power output get's removed soon.

My opinion: An overloaded AME should be able to put out significant amounts of power, at the risk of eventually exploding. With enough overloading, this will naturally lead to a smaller overloaded AME exceeding the output of a stable larger one. It gives engineering an interesting mechanic to move beyond the size restrictions of their AME, while giving a time limit on how long this is possible and a big punishment if they don't pay attention or get sabotaged.


tl;dr: I consider this working as intended.
